python/ray/serve/tests/test_deploy_2.py

```diff
@@ -238,9 +238,7 @@ def test_deployment_error_handling(serve_instance):
     def f():
         pass
 
-    with pytest.raises(
-        ValidationError, match="1 validation error for RayActorOptionsSchema.*"
-    ):
+    with pytest.raises(RuntimeError, match="The Deployment constructor failed"):
```

Comment:
Something related to "local working directory only supported per-job"  should still appear somewhere in the message, right?  Can we check for it with `match` or `traceback` (https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions)?